src/main/java/SendMail.java

```diff
@@ -0,0 +1,167 @@
+
+import com.google.gson.JsonParseException;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.lang.StringBuilder;
+
+import javax.mail.Address;
+
+
+
+
+public class SendMail {
+    /**
+     * Setup mail server SMTP connection and sends a mail using the context in Report and Mailserver objects.
+     * Takes in object of Report, and Mailserver
+     * See javax.mail for method documentation.
+     */
```

Comment:
This comment should probably be at the `SendMail` javadoc.

Reply:
Fixed.